### PR TITLE
documentation improvements for llm demo

### PR DESCRIPTION
**Pull request by dtrawins**
### 🧪 Checklist

- [ ] Unit tests added.
- [X] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: demos/continuous_batching/README.md [hunk 1]
```diff
@@ -53,7 +59,8 @@ node: {
   }
   node_options: {
       [type.googleapis.com / mediapipe.LLMCalculatorOptions]: {
-          models_path: "./"
+          models_path: "./",
+          cache_size: 50
```
Comment (mzegla) [inline demos/continuous_batching/README.md:63]: Should it be this big?

Comment (dtrawins) [reply demos/continuous_batching/README.md:63]: This value was used in benchmarking script and the results are included. Default value might be to low. Which value do you suggest?

Comment (mzegla) [reply demos/continuous_batching/README.md:63]: 16 maybe? For someone with 32 GBs RAM to be able to comfortably load it.

Comment (dtrawins) [reply demos/continuous_batching/README.md:63]: I added a not to reduce it if there is not enough RAM. I would expect on hosts with Xeon at least 128GB on board. 
Do you think this is good enough?

**Review by dkalinowski — APPROVED** [review]

**Review by mzegla — APPROVED** [review]

**Pull request merged**